src/Vatras/Framework/Definitions.agda

```diff
@@ -61,3 +61,22 @@ and hence expressions are parameterized in the type of this atomic data.
 -}
 𝔼 : Set₂
 𝔼 = 𝔸 → Set₁
+
+-- some default atoms
+module _ where
+  open import Data.String using (String; _≟_)
+
+  _String≟_ : DecidableEquality String
+  _String≟_ = _≟_
```

Comment:
Are these reexports really necessary? I would just define `STRING` and leave out `_String≟_`. This would also make the anonymous modules redundant.

Reply:
Well, I'm unsure about when these are actually necessary. Furthermore, with `import Data.String as String` you get the almost identical `String._≟_`.
However, this is really a nit pit only.

Reply:
Yeah, I guess we should remove them for now. We can later try to identify convenient notation when we actually need it.

Reply:
Maybe transforming our atom sets from the default sum type to a custom record would be senaible somewhen on the future, too.